models/UserManager.class.php

```diff
@@ -3,13 +3,45 @@
 class UserManager {
 	private $_user;
 
+    public function checkMail($mail) {
+        if (strlen($mail) > 30) {
+			return 'Email is too long !';
+        }
+        if (!filter_var($mail, FILTER_VALIDATE_EMAIL)) {
+			return 'Email is not valid';
+        }
+        if ($this->getByMail($mail)) {
+            return 'Email is already taken !';
+        }
+        return null;
+    }
+
+    public function checkPassword($password) {
+        if (strlen($password) > 20) {
```

Comment:
Same, i dont think we should limit the password size (or when he is too short btw.)

Reply:
lol non